src/plugins/index_pattern_field_editor/public/components/preview/field_list/field_list.tsx

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import React, { useState, useMemo, useCallback } from 'react';
+import VirtualList from 'react-tiny-virtual-list';
```

Comment:
Is virtualization absolutely necessary here? Accessibility of virtualization is usually quite bad. 

If it truly is, in EUI we've found [`react-window`](https://github.com/bvaughn/react-window/) to have the best experience and are using it for all virtualization there. 

So I think by using `react-window` you'd also avoid importing another library into Kibana.

Reply:
Yes we need virtualization as the list of field can grow to up to 5000+ fields. I checked in the Kibana dependencies and we are not using `react-window` but we do already have the `react-tiny-virtual-list` used in other places.

For the scope of this PR can we keep this lib and if we really don't want this lib anymore have a separate PR to remove it from Kibana and add the new `react-window` to the dependencies?